dxcint/poetry.lock

```diff
@@ -1,15 +1,15 @@
-# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
I used poetry 2.1.1 to generate this - not sure if there was reason to use 1.4.2 version.

Also, do we need to keep this file in the repository? @aogorodnikov-dna @r-i-v-a

Reply:
Note that cwl_runner does not have the poetry.lock saved to the github repo.